playwright/tests/customers.spec.ts

```diff
@@ -0,0 +1,215 @@
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const email = faker.internet.email();
+  const note = faker.lorem.sentence();
+  await customersPage.waitForNetworkIdle(() =>
+    customersPage.gotoCustomerDetailsPage(CUSTOMERS.editCustomer.id),
+  );
+  await customersPage.fillNote(note);
+  await customersPage.fillFirstAndLastName(firstName, lastName);
+  await customersPage.fillEmail(email);
+  await customersPage.saveCustomer();
+  await customersPage.successBanner.waitFor({ state: "visible" });
+  await expect(customersPage.pageHeader).toContainText(
+    `${firstName} ${lastName}`,
+  );
+  await expect(customersPage.customerNoteInput).toContainText(note);
+  await expect(customersPage.customerFirstNameInput).toHaveValue(firstName);
+  await expect(customersPage.customerLastNameInput).toHaveValue(lastName);
+  await expect(customersPage.customerEmailInput).toHaveValue(
+    email.toLowerCase(),
+  );
+});
+test("TC: SALEOR_202 Deactivate a customer @e2e @customer", async () => {
+  await customersPage.waitForNetworkIdle(() =>
+    customersPage.gotoCustomerDetailsPage(CUSTOMERS.customerToBeDeactivated.id),
+  );
+  await customersPage.customerActiveCheckbox.click();
+  await customersPage.saveCustomer();
+  await customersPage.successBanner.waitFor({ state: "visible" });
+  await expect(customersPage.customerActiveCheckbox).not.toBeChecked();
+});
+test("TC: SALEOR_203 Activate a customer @e2e @customer", async () => {
+  await customersPage.waitForNetworkIdle(() =>
+    customersPage.gotoCustomerDetailsPage(CUSTOMERS.customerToBeActivated.id),
+  ),
+    await customersPage.customerActiveCheckbox.click();
+  await customersPage.saveCustomer();
+  await customersPage.successBanner.waitFor({ state: "visible" });
+  await expect(customersPage.customerActiveCheckbox).toBeChecked();
+});
+
+test("TC: SALEOR_204 Delete customer from the details page @e2e @customer", async () => {
+  await customersPage.waitForNetworkIdle(() =>
+    customersPage.gotoCustomerDetailsPage(CUSTOMERS.deleteCustomer.id),
+  );
+  await customersPage.deleteCustomer();
+  await customersPage.deleteDialog.clickDeleteButton();
+  await customersPage.successBanner.waitFor({ state: "visible" });
+  await customersPage.waitForNetworkIdle(() =>
+    customersPage.goToCustomersListView(),
+  );
+  await customersPage.waitForNetworkIdle(() =>
+    customersPage.searchForCustomer(CUSTOMERS.deleteCustomer.email),
+  );
+  await expect(customersPage.emptyDataGridListView).toBeVisible();
+});
+test("TC: SALEOR_205 Bulk delete customers @e2e @customer", async () => {
+  const customersToBeBulkDeleted = CUSTOMERS.customersToBeBulkDeleted.names;
+
+  await customersPage.goToCustomersListView();
+  await customersPage.waitForNetworkIdle(() => customersPage.typeInSearchOnListView("bulk-delete"));
+  await customersPage.waitForDOMToFullyLoad();
+  const customers = [];
+  for (let i = 0; i < 3; i++) {
```

Comment:
This one deserve helper

Reply:
@wojteknowacki fixed https://github.com/saleor/saleor-dashboard/pull/4657/commits/0560f72238fb42e11de6b96d91f1570cbb37fe2d